src/component/app/TimelineRoutes.tsx

```diff
@@ -1,69 +1,51 @@
+import { useContext } from "react";
+import { Navigate, Route, Routes, useLocation, useParams } from "react-router";
+
 import { AuthRoute } from "component/app/AuthRoute";
+import LoadingBar from "component/loadingbar";
 import { Home } from "component/page/home";
 import { IVentureContext, VentureContext } from "context/VentureContext";
 import { useAuth } from "module/auth";
-import { calculateNamedSlug, getUniqueListBy } from "module/helpers";
+import { getUniqueListBy } from "module/helpers";
 import { useTimelinesByVentureId } from "module/hook/timeline";
-import { useVenturesBySlug } from "module/hook/venture";
-import { useContext } from "react";
-import { Navigate, Route, Routes, useLocation, useParams } from "react-router";
 import { ITimelineContext, TimelineContext } from "context/TimelineContext";
 import { useTimelineMembers } from "module/hook/user";
 import { useTimelineRole } from "module/hook/role";
 import { IUser } from "module/interface/user";
 import { IRole } from "module/interface/role";
-import LoadingBar from "component/loadingbar";
 
 export function TimelineRoutes() {
   const outerVentureContext = useContext(VentureContext);
-  const { ventureSlug = "" } = useParams();
+  const { ventureId, timelineId } = useParams();
```

Comment:
Improving the routes and params in `App`, `TimelineRoutes`, and `VentureRoutes`. There's a possibility I've introduced a bug, but everything seems fine in testing so far.